src/sdam/description/topology/mod.rs

```diff
@@ -766,6 +842,7 @@ impl Default for TransactionSupportStatus {
 /// Returned from `TopologyDescription::diff`.
 #[derive(Debug)]
 pub(crate) struct TopologyDescriptionDiff {
+    pub(crate) old_addresses: HashSet<ServerAddress>,
```

Comment:
`old_addresses` sort of implies "pre existing" addresses rather than removed ones. could we update this to something like `removed_addresses`? Feel free to change the `new_addresses` to like `added_addresses` if you think that would make this more consistent.

also see my other comment about expanding this to include "changed" or "updated" servers.